website/src/controllers/groups.js

```diff
@@ -137,7 +137,28 @@ api.get = function(req, res, next) {
   q.exec(function(err, group){
     if (err) return next(err);
     if (!group && gid!=='party') return res.json(404,{err: "Group not found or you don't have access."});
```

Comment:
sincerely I don't get why `&& gid!=='party'` is here but I would say it's out of the scope of this PR since it's been there for a long time :)